package.json

```
@@ -83,15 +84,16 @@
     "make-empty-github-commit": "^1.2.1",
     "netlify-cli": "^3.5.0",
     "patch-package": "^6.4.5",
-    "postcss": "^7",
+    "postcss": "8.4.12",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^2.0.5",
```

The reason prettier changed a few files is because your changes to the lockfile resulted in prettier being upgraded to `2.6.0`. Which is totally fine, but can you update this dep version? By removing the `^` we should no longer see random prettier issues like this unless we're explicitly updating it.

```suggestion
    "prettier": "2.6.0",
```

---

I'll do this @JessicaSachs